src/Administration/Resources/app/administration/src/module/sw-settings-country/component/sw-settings-country-general/sw-settings-country-general.html.twig

```diff
@@ -246,6 +246,33 @@
             />
             {% endblock %}
 
+            <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
+            {% block sw_settings_country_general_content_check_vat_id_format %}
```

Comment:
you should not add blocks to new admin js functionality

Reply:
(｡•́︿•̀｡)